Wrappers/Python/cil/utilities/quality_measures.py

```diff
@@ -18,36 +18,90 @@
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
 
 from cil.optimisation.functions import L2NormSquared, L1Norm
+from cil.framework import DataContainer
 import numpy as np
 
 
-def mse(dc1, dc2):    
+
+def mse(dc1, dc2, mask=None):    
     
     ''' Returns the Mean Squared error of two DataContainers
-    '''
     
-    diff = dc1 - dc2    
-    return L2NormSquared().__call__(diff)/dc1.size
+    Parameters
+    ----------
+    dc1: `DataContainer`
+        One image to be compared
+    dc2: `DataContainer`
+        Second image to be compared 
+    mask: array or `DataContainer` of Boolean values or 0's and 1's with the same dimensions as the `dc1` and `dc2`
```

Comment:
This should be a bit clearer. "The pixelwise operation only considers values where the mask is `True`.". This is the same with the functions below.

Reply:
Here I would specify the calculation is performed for `True` or Non zero values. It's not values of `1`  for non boolean datatypes as `where` will interpret 6 or 6.5 or -1 as `True`.

Reply:
Ah it was there, just not in the update I looked at!